installer/conf/logrotate.conf

```diff
@@ -6,3 +6,12 @@
     size 100M
     copytruncate
 }
+
+/var/opt/microsoft/omsagent/log/update_run_progress.log {
```

Comment:
Yes, you should use /%WORKSPACE_ID%/ here and test onboarding to make sure that your string (as well as Suju's in the above original block) gets replaced.